cmd/ocm/gcp/gcp-client-shim.go

```diff
@@ -607,13 +672,30 @@ func (c *shim) addPolicyBindingForProject(
 			return c.addMemberToBindingForProject(memberName, policy.Bindings[i])
```

Comment:
this I guess , can we replaced with applyFunc

Reply:
ty. updated.